src/hooks/useHasAppUpdated/useHasAppUpdated.ts

```diff
@@ -1,64 +1,49 @@
 import { useInterval } from '@chakra-ui/hooks'
 import axios from 'axios'
-import { useState } from 'react'
+import { useMemo, useState } from 'react'
+import { checkObjEquality } from 'lib/utils'
```

Comment:
can you please use `import isEqual from 'lodash/isEqual'` here rather than a hand rolled util?

https://lodash.com/docs/4.17.15#isEqual

Reply:
yes, of course! I didn't realize you were using lodash. 
I change it fast! 
Thank u!

Reply:
it's done!